### PR TITLE
PE-3044: Redesign feedbacks implementations

### PR DESCRIPTION
**Pull request by thiagocarvalhodev**
--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/4datasaeb0k28
iOS release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:ios:06ea33a95a2e0d42ffce07/releases/6blot6bj5nc40

### CODE REVIEW

File: lib/authentication/ardrive_auth.dart [hunk 1]
```diff
@@ -18,18 +18,12 @@ abstract class ArDriveAuth {
...
-  //   String password,
-  //   ProfileType profileType, {
-  //   required String localizedReason,
-  // });
   Future<User> unlockUser({required String password});
   Future<void> logout();
   User? get currentUser;
   Stream<User?> onAuthStateChanged();
   Future<bool> isBiometricsEnabled();
+  bool isOwner(String walletAddress);
```
Comment (karlprieb) [inline lib/authentication/ardrive_auth.dart:26]: move this utility to another place, let's keep auth simple

File: lib/main.dart [hunk 2]
```diff
@@ -144,147 +147,188 @@ class AppState extends State<App> {
...
+
+  for (String assetPath in assetPaths) {
+    precacheImage(
+      AssetImage(assetPath),
+      context,
+    );
+  }
+}
+
+void preCacheOnBoardingAssets(BuildContext context) {
```
Comment (karlprieb) [inline lib/main.dart:318]: move to utils

File: lib/main.dart [hunk 3]
```diff
@@ -52,6 +53,8 @@ late TurboService _turbo;
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
 
+  // await precacheIcons();
```
Comment (karlprieb) [inline lib/main.dart:56]: remove

File: lib/main.dart [hunk 4]
```diff
@@ -144,147 +147,188 @@ class AppState extends State<App> {
...
+            },
           ),
         ),
-      );
+      ),
+    );
+  }
+}
+
+void preCacheLoginAssets(BuildContext context) {
```
Comment (karlprieb) [inline lib/main.dart:302]: move to utils

File: pubspec.yaml [hunk 5]
```diff
@@ -108,7 +108,7 @@ dev_dependencies:
     sdk: flutter
   flutter_driver:
     sdk: flutter
-  test: 1.21.4
+  test:
```
Comment (karlprieb) [inline pubspec.yaml:111]: ?

File: lib/main.dart (hunk 4 above)
Comment (thiagocarvalhodev) [reply lib/main.dart]: https://github.com/ardriveapp/ardrive-web/pull/1029/commits/f8a27819c8fb60f14793733932942d02a811c738

File: lib/main.dart (hunk 3 above)
Comment (thiagocarvalhodev) [reply lib/main.dart]: https://github.com/ardriveapp/ardrive-web/pull/1029/commits/f8a27819c8fb60f14793733932942d02a811c738

File: lib/main.dart (hunk 2 above)
Comment (thiagocarvalhodev) [reply lib/main.dart]: https://github.com/ardriveapp/ardrive-web/pull/1029/commits/f8a27819c8fb60f14793733932942d02a811c738

File: pubspec.yaml (hunk 5 above)
Comment (thiagocarvalhodev) [reply pubspec.yaml]: https://github.com/ardriveapp/ardrive-web/pull/1029/commits/289b23fc7c22515955229fa01b5ca588f8c22f19

File: lib/authentication/ardrive_auth.dart (hunk 1 above)
Comment (thiagocarvalhodev) [reply lib/authentication/ardrive_auth.dart]: https://github.com/ardriveapp/ardrive-web/pull/1029/commits/be8efc685525bf03ee7dc7680faa0cdf871540ec

**Review by karlprieb — APPROVED** [review]

**Pull request merged**